app/src/ui/create-branch/create-branch-dialog.tsx

```diff
@@ -422,8 +432,28 @@ export class CreateBranch extends React.Component<
         this.state.startPoint === StartPoint.UpstreamDefaultBranch
           ? this.state.startPoint
           : StartPoint.CurrentBranch
+      return (
+        <div>
+          {this.renderOptions(items, selectedValue)}
+          {this.renderForkLink()}
+        </div>
+      )
+    }
+  }
 
-      return this.renderOptions(items, selectedValue)
+  private renderForkLink = () => {
+    if (isRepositoryWithForkedGitHubRepository(this.props.repository)) {
+      return (
+        <div className="secondary-text">
+          Your default branch source will be determined by your{' '}
+          <LinkButton onClick={this.onForkSettingsClick}>
```

Comment:
This link is not displaying inline and seems to wrap oddly. Think we may want to add a css rule to the `.link-button-component` to be `display: inline;` when used in the dialog.

Reply:
added 👍🏻